packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -457,6 +447,16 @@ export class AppStudioPluginImpl {
       `${ctx.root}/${BuildFolderName}/${AppPackageFolderName}/manifest.` +
       (isLocalDebug ? "local" : ctx.envInfo.envName) +
       `.json`;
+    if (!(await fs.pathExists(manifestFileName))) {
```

Comment:
Can this condition be executed since user will be blocked at the above AppStudioError.GetRemoteConfigError when isProvisionSucceeded is false?

Reply:
For SPFx project, it will goes here.